wormhole-connect/src/components/TokensModal.tsx

```diff
@@ -260,6 +315,20 @@ function TokensModal(props: Props) {
     }
   };
 
+  const handleMoreTokens = (href: string, target: string = '_self') => {
+    let hydratedHref = href;
+    if (fromChain) {
+      hydratedHref = hydratedHref.replace('{:sourceChain}', fromChain);
+    }
+    if (toChain) {
+      hydratedHref = hydratedHref.replace('{:targetChain}', toChain);
+    }
+    window.open(
+      hydratedHref.replace('&targetChain={:targetChain}', ''),
```

Comment:
the previous `if` populates the toChain value if it is present, otherwise, we want to remove the entire placeholder, and that is why it is replacing with empty string